File: Typescript/Practice/easy/easy_filterString.ts
#!/usr/bin/env npx ts-node

function filterArray (arr: Array<string | number>) {
  let arr2 = new Array();
  for (let i of arr) {
    if (typeof i !== 'string') {
      arr2.push(i);
    };
  };
  return arr2;
};

console.log(filterArray([1, 2, "a", "b"]));
console.log(filterArray([1, "a", "b", 0, 15]));
console.log(filterArray([1, 2, "aasf", "1", "123", 123]));

File: Typescript/Practice/26072022/easy/easy_filterString.py
#!/usr/bin/env python

def filter_list(list1):
  list2 = []
  for i in list1:
    if type(i) != str:
      list2.append(i)
  return list2

print(filter_list([1, 2, "a", "b"]))
print(filter_list([1, "a", "b", 0, 15]))
print(filter_list([1, 2, "aasf", "1", "123", 123]))
